eca-chat.el

```diff
@@ -14,6 +14,8 @@
 (require 'f)
 (require 'markdown-mode)
 (require 'compat)
+(require 'ediff)
+(require 'smerge-mode)
```

Comment:
This obligates users to have these packages installed, which is no good, we should make them optional, requiring if available and providing the feature if available